### ISSUE TITLE
Nuclear pore sumoylation annotations

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3658]
Hi @deustp01 ,
All of the nuclear pore c omplex appears to be annotated to sumoylation by Reactome- seems odd?
Val

NUP160 | Nuclear pore complex protein Nup160 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup160 pthr21286 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP85 | Nuclear pore complex protein Nup85 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | frount protein-related pthr13373 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP133 | Nuclear pore complex protein Nup133 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup133 pthr13405 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP93 | Nuclear pore complex protein Nup93 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup93 nucleoporin nup93 dead eye protein pthr11225 | protein |   | Reactome:R-HSA-4655355 | 20181121
NUP205 | Nuclear pore complex protein Nup205 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup205 pthr31344 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP88 | Nuclear pore complex protein Nup88 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin nup84-related pthr13257 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP210 | Nuclear pore membrane glycoprotein 210 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore membrane glycoprotein gp210-related pthr23019 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP54 | Nucleoporin p54 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin p54 pthr13000 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | RAE1 | mRNA export factor |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | mrna export factor and bub3 pthr10971 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP42 | Nucleoporin NUP42 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin-like protein 2 pthr46527 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP50 | Nuclear pore complex protein Nup50 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | ran binding protein pthr23138 | protein |   | Reactome:R-HSA-4655355 | 20181121
NUP58 | Nucleoporin p58/p45 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin p58/p45 nucleoporin-like protein 1 pthr13437 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP188 | Nucleoporin NUP188 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin nup188 homolog pthr31431 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | POM121C | Nuclear envelope pore membrane protein POM 121C |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup pthr23193 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP214 | Nuclear pore complex protein Nup214 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup pthr23193 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | SENP5 | Sentrin-specific protease 5 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | sentrin/sumo-specific protease pthr12606 | protein |   | Reactome:R-HSA-3065679 | 20181122
  | POM121 | Nuclear envelope pore membrane protein POM 121 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup pthr23193 | protein |   | Reactome:R-HSA-4655355 | 20181121
NUP155 | Nuclear pore complex protein Nup155 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein nup155 pthr10350 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP98 | Nuclear pore complex protein Nup98-Nup96 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin pthr23198 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP107 | Nuclear pore complex protein Nup107 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nup107-related pthr13003 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | SAE1 | SUMO-activating enzyme subunit 1 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | ubiquitin-activating enzyme e1 pthr10953 | protein |   | Reactome:R-HSA-3065676 | 20181122
  | SAE1 | SUMO-activating enzyme subunit 1 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | ubiquitin-activating enzyme e1 pthr10953 | protein |   | Reactome:R-HSA-3065678 | 20181122
  | NUP62 | Nuclear pore glycoprotein p62 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore glycoprotein p62-related pthr12084 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | SEH1L | Nucleoporin SEH1 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nuclear pore complex protein sec13 / seh1 family member pthr11024 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP35 | Nucleoporin NUP35 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin nup35 pthr21527 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP37 | Nucleoporin Nup37 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin nup37 p37 -related pthr22806 | protein |   | Reactome:R-HSA-4655355 | 20181121
  | NUP43 | Nucleoporin Nup43 |   | protein sumoylation |   | Reactome | Homo sapiens | TAS |   | nucleoporin nup43 pthr22652 | protein |   | Reactome:R-HSA-4655355 | 20181121

**Comment by deustp01** [comment]
I think that is happening because RanBP2, the gene product that directly enables the SUMOylation, occurs as a component of a complex with all of these other proteins, the complex was (correctly) identified as the catalyst of SUMOylation, but the curator did not take advantage of the option to identify one of the components of the complex as its active unit. I have now fixed that and the change should propagate to our public site with our next release in about a month. (I also though that when we assigned a catalysis molecular function to a heteromultimer, unless we also specified an active unit, no part of the complex was associated with the GO MF term in our GAF. I will ask the developers to try to figure out why this one escaped.
